a3p-integration/proposals/a:upgrade-14/test.sh

```diff
@@ -2,3 +2,7 @@
 source /usr/src/upgrade-test-scripts/env_setup.sh
 
 yarn ava
+
+npm install --global tsx
+
+./testRepairs.ts
```

Comment:
this script would make sense as an Ava test file. probably not worth refactoring at this point tho

Reply:
I couldn't resist: c45e770e5002bf4a1df89383e806c877a5bcddf4

Reply:
I adopted that. Thanks. Makes more sense than `step()`.